src/lib.rs

```diff
@@ -1,6 +1,14 @@
+//! # HydraDX Math
+//!
+//! A collection of utilities to make performing liquidity pool
+//! calculations more convenient.
+
 #![cfg_attr(not(feature = "std"), no_std)]
 
+/// !! Insert Description here. !!
```

Comment:
So lets insert something here.

Reply:
Not so sure what to insert, i'd rather not insert
"A collection of utilities..." a third time :D

Reply:
so just delete it.